### PR TITLE
feat: Add Avro Container File output support

### PR DESCRIPTION
**Pull request by jjaakola-aiven**

### CODE REVIEW

File: src/main/java/io/aiven/kafka/connect/s3/S3SinkTask.java [hunk 1]
```diff
@@ -204,7 +205,8 @@ private String oldFullKey(final SinkRecord record) {
                     record, VariableTemplatePart.Parameter.of("padding", "true")
                 )
             );
-        return prefix + key + config.getCompressionType().extension();
+        final String formatSuffix = FormatType.AVRO.equals(config.getFormatType()) ? ".avro" : "";
+        return prefix + key + formatSuffix + config.getCompressionType().extension();
```
Comment (jeqo) [inline src/main/java/io/aiven/kafka/connect/s3/S3SinkTask.java:209]: nit: we may add a reminder to keep this aligned with `AivenCommonConfig#getFilename`

**Review by jeqo — DISMISSED** [review]
Just a minor comment, otherwise looks great.

File: src/main/java/io/aiven/kafka/connect/s3/S3SinkTask.java (hunk 1 above)
Comment (jjaakola-aiven) [reply src/main/java/io/aiven/kafka/connect/s3/S3SinkTask.java:210]: Done.

**Review by jeqo — APPROVED** [review]
LGTM

**Pull request merged**